src/Controls/src/Core/Items/ItemsView.cs

```diff
@@ -113,15 +114,50 @@ public int RemainingItemsThreshold
 
 		internal static readonly BindableProperty InternalItemsLayoutProperty =
 			BindableProperty.Create(nameof(ItemsLayout), typeof(IItemsLayout), typeof(ItemsView),
-				LinearItemsLayout.Vertical, propertyChanged: OnInternalItemsLayoutPropertyChanged);
+				defaultValueCreator: CreateDefaultItemsLayout, propertyChanged: OnInternalItemsLayoutPropertyChanged);
+
+		static IItemsLayout CreateDefaultItemsLayout(BindableObject bindable)
+		{
+			var layout = new LinearItemsLayout(ItemsLayoutOrientation.Vertical);
+
+			if (layout is BindableObject bo && bindable is ItemsView itemsView)
+			{
+				SetInheritedBindingContext(bo, itemsView.BindingContext);
+				bo.PropertyChanged += itemsView.OnItemsLayoutPropertyChanged;
```

Comment:
so no need to unsubscribe here?

Reply:
Don t we do it on OnInternalItemsLayoutPropertyChanged ? the subscribe and unsubscribe?